photon-ml/src/main/scala/com/linkedin/photon/ml/function/glm/PoissonLossFunction.scala

```diff
@@ -51,12 +44,10 @@ object PointwisePoissonLossFunction extends PointwiseLossFunction {
 
   /**
    * d^2^l/dz^2^ = exp(z)
-   * 
+   *
    * @param margin The margin, i.e. z in l(z, y)
    * @param label The label, i.e. y in l(z, y)
    * @return The value and the 2st derivative with respect to z
    */
-  override def d2lossdz2(margin: Double, label: Double): Double = {
```

Comment:
@ferd36 As per our offline talk, I'll make this change, and log an issue related to refactoring the relationships between `TRON`, `TwiceDiffFunction`, and `PointwiseLossFunction` (and derived classes).